blueocean-pipeline-api-impl/src/main/java/io/jenkins/blueocean/rest/impl/pipeline/PipelineRunImpl.java

```diff
@@ -155,13 +154,9 @@ public void stop() {
 
     @Exported(name = "commitId")
     public String getCommitId() {
-        BuildData data = run.getAction(BuildData.class);
-
+        SCMRevisionAction data = run.getAction(SCMRevisionAction.class);
         if (data != null){
-            Revision revision = data.getLastBuiltRevision();
-            if(revision != null){
-                return revision.getSha1String();
-            }
+            return data.getRevision().toString();
```

Comment:
@jglick in that case what's the right thing to use? How can we get checked out repo's commit hash?